.cicd/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
```

Comment:
It looked like in eos-vm, the plan was to move away from helpers. No?  This is not a requirement to change, but would consider flattening the .cicd directory a little to encourage keeping it simple going forward.

* The installation-build.sh could be put into the eosio-installation-images.yml directly
* The docker-hash.sh, general.sh, and clang.make seems to make sense because they reduce code duplication, but they don't really need to be in a subdirectory.

Reply:
We're actually able to delete eosio-installation-images.yml and base-images.yml completely which helps. I moved installation-build.sh out into the root since it's not really a helper.

I believe the pattern we've been using for all repos is to keep helpers, dockerfiles, and anything else under directories. Here are some of the reasons I think we should stick to folders:

1. Let's say we delete test-metrics. If it's all contained to a folder, we just delete the folder. This prevents us from missing a file and leaving stuff laying around which isn't used anymore.
2. Visually, it's better looking and easier to find things. If I need to find and replace stuff in dockerfiles,  they're all in docker/, versus spread out in the main directory.